buildSrc/src/main/java/org/elasticsearch/gradle/internal/test/DistroTestPlugin.java

```diff
@@ -364,7 +370,7 @@ private static void configureVMWrapperTasks(
         List<ElasticsearchDistribution> currentDistros = new ArrayList<>();
 
         for (Architecture architecture : Architecture.values()) {
-            Arrays.stream(Type.values()).filter(t -> t != Type.INTEG_TEST_ZIP && t != Type.ARCHIVE).forEach(type -> {
+            List.of(DEB, RPM, DOCKER, DOCKER_IRONBANK, DOCKER_UBI).stream().forEach(type -> {
```

Comment:
We should probably move this list into `InternalElasticsearchDistributionTypes` that way when we add a new one we don't risk forgetting to update this.

Reply:
Will do